php-checks/src/main/java/org/sonar/php/checks/security/QueryUsageCheck.java

```diff
@@ -169,10 +169,7 @@ private boolean isSuspiciousConcat(BinaryExpressionTree tree) {
     while (!isSuspicious && !operands.isEmpty()) {
       ExpressionTree operand = operands.pop();
       switch (operand.getKind()) {
```

Comment:
This could also be rewritten with an enhanced switch statement to avoid `break`s.
Edit: OK, I see that it would create an empty branch then, so can be left as is